### PR TITLE
Add right rail only theme

### PR DESCRIPTION
**Pull request by debugwand**
Required for App where we never have use for a left hand rail and it is making the layout weird in tablet landscape and up

### CODE REVIEW

File: scss/_mixins.scss [hunk 1]
```diff
@@ -61,6 +61,10 @@
 	.o-topper--news-package {
 		@include oTopperThemeNewsPackage;
 	}
+
+	.o-topper--right-rail {
+		@include oTopperThemeRightRail;
```
Comment (adgad) [inline scss/_mixins.scss]: Not related - but there's a `o-topper--centered` which for some reason is in `oTopperElement`. Could we make that a theme too? (it's effectively the same sort of thing, a different grid column setup)

Comment (debugwand) [reply scss/_mixins.scss]: sure

File: scss/themes/_centered.scss [hunk 2]
```diff
@@ -0,0 +1,5 @@
+// sass-lint:disable mixins-before-declarations
+
+@mixin oTopperThemeRightRail {
```
Comment (adgad) [inline scss/themes/_centered.scss]: copy pasta?

Comment (debugwand) [reply scss/themes/_centered.scss]: yes, rebased fix done and in test now

**Pull request merged**